docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,37 +371,41 @@ which is a CDI bean and can be ``@Inject``ed.
 [[jdbcstore]]
 == Configure storing of Quarkus transaction logs in a database
 
-In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
+In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a Java Database Connectivity (JDBC) datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+However, in cloud-native apps, it is important to assess transactions after careful evaluation.
+The `narayana-jta` extension, which manages these transactions, requires stable storage, a unique reusable node identifier, and a steady IP address to work correctly.
+While the JDBC object store provides a stable storage, users must still plan how to meet the other two requirements.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+Quarkus then allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store._<property>_` properties, where _<property>_ can be:
```

Comment:
Removing the `IMPORTANT` keyword at (the old) line number 372, it seems to me that this sentence here becomes detached from the flow of the document. In fact, I didn't follow what `then` was referring to.

Reply:
I guess "then" is meant as "after you assessed whether an in-database transaction store is right for you", @MichalMaler ?

It's probably something that would solve itself once we resolve the conversation above about "assess transactions after careful evaluation".

Reply:
New wording LGTM:

> Quarkus, after you evaluated whether an in-database transaction store is right for you, allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store._<property>_` properties, where _`<property>`_ can be: